### ISSUE TITLE
Tooltips for exported properties not showing up

### ISSUE THREAD

**Issue by inhalt120g** [godotengine/godot#100910]
### Tested versions

v4.3.stable.official [77dcf97d8]
(tried occasionally here and there from 4.0 onwards but was never able to make it work)

### System information

Newest MacOS on Mac Mini M1

### Issue description

I tried all possible combinations following various older discussions on this topic, but no matter what… the tooltips are not showing up on my end. I know they work for some people but even the simplest setup does not work on my end:
![Image](https://github.com/user-attachments/assets/0f8f5485-acae-4752-8f09-026e9d7cd6ae)

### Steps to reproduce

Copy and paste the code below, and check whether any tooltips appear when you hover over the variables.

### Minimal reproduction project (MRP)

```
## A short explanation 1
@export var my_float1 : float

## A short explanation 2
@export var my_float2 : float = 0
```

**Comment by Saulo-de-Souza** [comment]
I tested on Godot 4.4 dev 7 and stable 4.3. 
I copied the same code you provided and it didn't appear until I pressed CRTL + S (save). 
Did you save the file?

**Comment by inhalt120g** [comment]
I saved, switched to another node and then back to the node with the tooltips, restarted the editor, restarted Godot, tried different random things like removing space after `##`, tried splitting the comment into two lines, and other stuff I forgot. Here is the result:
![Image](https://github.com/user-attachments/assets/5fe068f5-96ac-46c5-aac6-9ab28dcafad5)

**Comment by dalexeev** [comment]
* Probably a duplicate of #72952, please see if #95821 fixes this.

**Comment by inhalt120g** [comment]
@dalexeev How do I check it in v4.3.stable.official [77dcf97d8]? Or am I supposed to download the 4.4 build?

**Comment by dalexeev** [comment]
@inhalt120g No, this fix is ​​not included in 4.3 or 4.4 dev 7, as the PR has not been merged yet. If it's not too much trouble for you, you can compile the engine from source, after applying the patch from the above PR. Or you can download the artifact from the Actions tab (note that GitHub removes old artifacts after a while). See the [documentation](https://docs.godotengine.org/en/stable/contributing/workflow/testing_pull_requests.html) for details.

**Comment by inhalt120g** [comment]
I'd like to, but… that's way too technical for me (^.^);;;

**Comment by pewcworrell** [comment]
I was having this issue. Bread crumbs lead me here, and I made a cherry pick of the suggested solution and tried this out in our project. I now have tooltips! Thank you.
